README.md

```diff
@@ -1 +1,171 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+This Project was buiilt with React and Redux
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **API Integration**
+- **Displays data in Real time**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/7b0db8a8f9824a1fa52a5ffa9c440acf)
+- [Live site Link](https://lively-platypus-c3877d.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- React and Redux linters
+
+In order to run this project you need:
+
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+
+
+### Install
+
+Install this project with:
+npm
+Example command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+npm start or live server
+Example command:
+
+```sh
+  GitHub Pages Server
+```
+
+### Run tests
+
+To run tests, run the following command:
+npm test
+Example command:
+
+```sh
+  npx eslint "**/*.{js,jsx}"
+  npx stylelint "**/*.{css,scss}"
+```
+
+### Deployment
+
+It will be deployed after merge
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+ **Isabella Otoo**
+
+- GitHub: [@githubhandle](https://github.com/Bellagirl-maker)
+- Twitter: [@twitterhandle](https://twitter.com/isabella_otoo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/isabella-otoo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **making it responsive**
+- [ ] **Design for mobile First**
+- [ ] **Making it dynamic**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+if you like this project, kindly give it 5 stars.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my colleagues and code reviewers in microverse for helping me.
+This capstone project is based on the original design idea of <a href="https://www.behance.net/sakwadesignstudio>Nelson Sakwa on Behance.</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Please make sure that the MIT file is there and working properly when this link is clicked.

Reply:
@ShadyShawkat done